### ISSUE TITLE
Arduino nano to RPi?

### ISSUE THREAD

**Issue by jsiegmund** [Jark/FTDISample#6]
Not really an issue I guess, but I would very much appreciate some help! I've got an RPi running Win10 IoT and an Arduino nano I was looking to use to gather some analog sensor data. All samples work great when the nano is connected to my PC, but the Pi doesn't recognize the device. Thought I would use the same trick to install the FTDI driver, but nothing happens. Connected devices on the Pi doesn't list it, nor won't the sample find anything. I checked VID / PID and added this to the appmanifest:

`<!--My Custom Device-->`
`<Device Id="vidpid:1A86 7523">`
`<Function Type="name:vendorSpecific" />`
`</Device>`

Still nothing. Any ideas? Should this be working in the first place, or am I trying something that is a no-go?

**Comment by djaus2** [comment]
I have connected and used a Uno and Leonard with my RPI2. Have just released a new project on Codeplex that uses it. See my blogs http://embedded101.com/Blogs/David-Jones
You connect the micro USB from the Arduino device to USB host on the RPI2.
This interface isn't FTDI but works "out-of-the-box".with Windows 10 IoT Core.
I have discussed the Serial options on my blog.
Use the "Official" IoT Sample serial example on GitHub not this FTDI Sample.
I assume that the USB device interface on the nano is the same as that on my two Arduino devices.

**Comment by jsiegmund** [comment]
Hmm not sure what I'm missing here. The official serial example shows one device whether the nano is connected or not. Looks something like "\?\ACPI#MSFT8000#1#{guid}. Since that one shows also when it's not connected, gonna guess that's not the one. 

I checked the nano's hardware id's when connected to my pc, list as:
USB\VID_1A86&PID_7523&REV_0254
USB\VID_1A86&PID_7523

Nothing more. The device itself is called "USB-SERIAL CH340 (COM4)". Maybe this is the right time to mention that this is a Chinese clone thingy and not the real nano. I was perfectly able to use the Arduino software to upload stuff from my PC though, which I did to upload the Firmata software. Maybe that's where I took the wrong turn? 

Sorry for being a bit noobish here. I'm really in the dark with this stuff and you seemed like a proper pro :)
